exams/pearson.py

```diff
@@ -141,6 +149,19 @@ def _profile_country_to_alpha3(profile):
 ], field_prefix='profile')
 
 
+write_ead_file = _tsv_writer([
+    ('AuthorizationTransactionType', 'operation'),
+    ('ClientAuthorizationID', 'id'),
+    ('ClientCandidateID', 'user.profile.student_id'),
+    ('ExamSeriesCode', 'course.program.exam_series_code'),
+    ('Modules', 'course.exam_module'),
+    ('Accommodations', lambda _: ''),
+    ('EligibilityApptDateFirst', lambda exam_auth: _format_date(exam_auth.date_first_eligible)),
+    ('EligibilityApptDateLast', lambda exam_auth: _format_date(exam_auth.date_last_eligible)),
+    ('LastUpdate', lambda exam_auth: _format_datetime(exam_auth.updated_on)),
+])
```

Comment:
I think you should move this code inside a function like 
```
def get_ead_file_writer(<args>):
    return _tsv_writer([
         ...
```
In this way it is a bit more clear that you get a configured writer. Plus I am not convinced that this code should be executed at import time. 
If I am missing something let me know.